parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetFileReader.java

```diff
@@ -566,7 +581,10 @@ public ColumnChunkPageReader readAllPages() throws IOException {
                     this.readAsBytesInput(compressedPageSize),
                     dataHeaderV1.getNum_values(),
                     uncompressedPageSize,
-                    fromParquetStatistics(dataHeaderV1.getStatistics(), descriptor.col.getType()),
+                    fromParquetStatistics(
+                        fileMetaData.getCreatedBy(),
```

Comment:
this isn't guarded by a check for fileMetaData being null, but it should be, right? (use of the deprecated constructor).